test/jdk/java/awt/print/PrinterJob/PrintLatinCJKTest.java

```diff
@@ -21,57 +21,72 @@
  * questions.
  */
 
-/**
+/*
  * @test
- * @bug 800535 8022536
+ * @bug 8022536
+ * @library ../../regtesthelpers
+ * @build PassFailJFrame
  * @summary JDK7 Printing: CJK and Latin Text in string overlap
- * @run main/manual=yesno PrintLatinCJKTest
+ * @run main/manual PrintLatinCJKTest
  */
 
+import java.awt.BorderLayout;
 import java.awt.Font;
 import java.awt.Graphics;
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
 import java.awt.print.PageFormat;
-import java.awt.print.Pageable;
 import java.awt.print.Printable;
 import java.awt.print.PrinterException;
 import java.awt.print.PrinterJob;
+import java.lang.reflect.InvocationTargetException;
 import javax.swing.JButton;
 import javax.swing.JFrame;
-import javax.swing.JTextArea;
 
-import javax.swing.SwingUtilities;
+import static javax.swing.SwingUtilities.invokeAndWait;
+
+public class PrintLatinCJKTest implements Printable {
 
-public class PrintLatinCJKTest implements Printable, ActionListener {
+    private static PrintLatinCJKTest testInstance = new PrintLatinCJKTest();
+    private static JFrame frame;
+    private static String info = """
+            To test 8022536, if a remote printer is the system default,
+            it should show in the dialog as the selected printer.
```

Comment:
Should these two lines be moved to the bottom of the instructions? This case is not the main test case of this test and it's also covered by another test `javax/print/TextFlavorTest.java`.